ksql-rest-app/src/test/resources/ksql-plan-schema/schema-final.json

```diff
@@ -0,0 +1,885 @@
+          "type" : "array",
+          "items" : {
+            "type" : "string"
+          }
+        },
+        "sink" : {
+          "type" : "string"
+        },
+        "queryId" : {
+          "type" : "string"
+        },
+        "physicalPlan" : {
+          "$ref" : "#/definitions/ExecutionStep"
+        }
+      },
+      "required" : [ "sources", "sink", "queryId", "physicalPlan" ]
+    },
+    "StreamAggregate" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "streamAggregateV1" ],
+          "default" : "streamAggregateV1"
+        },
+        "properties" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/DefaultExecutionStepProperties"
+          } ]
+        },
+        "source" : {
+          "$ref" : "#/definitions/ExecutionStep"
+        },
+        "formats" : {
+          "$ref" : "#/definitions/Formats"
+        },
+        "nonFuncColumnCount" : {
+          "type" : "integer"
+        },
+        "aggregations" : {
+          "type" : "array",
+          "items" : {
+            "type" : "string"
+          }
+        }
+      },
+      "title" : "streamAggregateV1",
+      "required" : [ "@type", "properties", "source", "formats", "nonFuncColumnCount", "aggregations" ]
+    },
+    "DefaultExecutionStepProperties" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "default" ],
+          "default" : "default"
+        },
+        "queryContext" : {
+          "type" : "string"
+        }
+      },
+      "title" : "default",
+      "required" : [ "@type", "queryContext" ]
+    },
+    "StreamFilter" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "streamFilterV1" ],
+          "default" : "streamFilterV1"
+        },
+        "properties" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/DefaultExecutionStepProperties"
+          } ]
+        },
+        "source" : {
+          "$ref" : "#/definitions/ExecutionStep"
+        },
+        "filterExpression" : {
+          "type" : "string"
+        }
+      },
+      "title" : "streamFilterV1",
+      "required" : [ "@type", "properties", "source", "filterExpression" ]
+    },
+    "StreamFlatMap" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "streamFlatMapV1" ],
+          "default" : "streamFlatMapV1"
+        },
+        "properties" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/DefaultExecutionStepProperties"
+          } ]
+        },
+        "source" : {
+          "$ref" : "#/definitions/ExecutionStep"
+        },
+        "tableFunctions" : {
```

Comment:
Would be good to have a similar naming convention for `tableFunctions` here and `aggregations` in the aggregation steps